packages/cli/src/components/hooks/usePreviewHotkeys.tsx

```diff
@@ -7,50 +7,56 @@ type Options = {
 };
 
 export const hotkeysMap = {
-  showPreviews: '/',
-  viewModeDesktop: '1',
-  viewModeHTML: '3',
-  viewModeMobile: '2',
-  viewModeNext: ']',
-  viewModePrevious: '[',
+  showPreviews: "/",
+  viewModeDesktop: "1",
+  viewModeHTML: "3",
+  viewModeMobile: "2",
+  viewModeNext: "]",
+  viewModePrevious: "[",
 };
 
-const viewModeOrder: ViewMode[] = ['desktop', 'mobile', 'html']
+const viewModeOrder: ViewMode[] = ["desktop", "mobile", "html"];
```

Comment:
Are you using Prettier? That would prevent formatting from appearing in diffs. When I made this changes nothing formatted the code for me 😔

Reply:
yes, this is formatted by prettier

Reply:
How do you run Prettier @psugihara?

Reply:
i have it auto-formatting on save in my vscode